src/meta/ClusterManager.cpp

```diff
@@ -0,0 +1,90 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+
+#include "base/Base.h"
+#include "fs/FileUtils.h"
+#include "ClusterManager.h"
+
+namespace nebula {
+namespace meta {
+
+bool ClusterManager::loadOrCreateCluId() {
+    if (loadClusterId()) {
+        return true;
+    }
+    createClusterId();
+    return dumpClusterId();
+}
+
+
+bool ClusterManager::loadClusterId() {
+    int fd = ::open(clusterIdPath_.c_str(), O_RDONLY);
+    if (fd < 0) {
+        LOG(WARNING) << "loadClusterId failed!";
+        return false;
+    }
+    char buff[32] = {};
+    int len = ::read(fd, buff, sizeof(buff));
+    if (len >= 0) {
+        buff[len] = '\0';
+    } else {
+        buff[0] = '\0';
+    }
+
+    ClusterID clusterId = folly::to<int64_t>(buff);
+    LOG(INFO) << "int clusterId: " << clusterId;
+    if (clusterId == 0) {
+        return false;
+    } else {
+        clusterId_ = clusterId;
+        clusterIsSet_ = true;
+        LOG(INFO) << "loadClusterId succeed! clusterId_: " << clusterId_;
+        return true;
+    }
+    ::close(fd);
```

Comment:
close fd before return?

Reply:
fixed!

Reply:
the code can not execute here, so can delete this close.

Reply:
yes